Api/Health Checks.md

```diff
@@ -0,0 +1,125 @@
+ASP.NET Core offers the built-in Health Checks feature for monitoring and reporting the health and availability of our application infrastructure components. Health checks provide a quick and automated way to verify that our application’s dependencies and components are running smoothly. They help us proactively identify issues within our application, allowing us to address them before they impact users. Regularly verifying the health of our application components can ensure a more reliable and resilient system.
+
+## Basic Health Check
+
+For many apps, a basic health probe configuration that reports the app's availability to process requests (liveness) is sufficient to discover the app's status.
+
+The basic health probe is very simple to implement. All we have to do is register health check services and configure the Health Checks Middleware to respond at a URL endpoint with a health response in `Program.cs`: 
+
+```c#
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddHealthChecks();
+
+var app = builder.Build();
+
+app.MapHealthChecks("/api/health");
+
+app.Run();
+```
+
+After adding these, we can access our application on `"/api/health"` and the application will respond with one of the three distinct `HealthStatus` values:
+* `HealthStatus.Healthy`
+* `HealthStatus.Degraded`
+* `HealthStatus.Unhealthy`
```

Comment:
With this default setup, in what cases will the response be anything other than `Healthy`?
Also, since we're listing them here, it would be nice to have a short explanation of what each one means - the first one is obvious, but what are the differences between Degraded and Unhealthy?

Reply:
Right, I missed this. With the basic setup the only possible response is `Healthy`, and the other ones can be configured with custom health checks. I added this information and short explanations of each status so feel free to check them out :)